diff --git a/DESCRIPTION b/DESCRIPTION
index 31154a0..bf5fcf0 100644
--- a/DESCRIPTION
+++ b/DESCRIPTION
@@ -13,6 +13,6 @@ Description: Infer and discriminate RIP peaks from RIP-seq alignments
         post-alignments processing to visualization and annotation.
 Depends: R (>= 2.15), methods, S4Vectors (>= 0.9.25), IRanges, GenomicRanges,
-        SummarizedExperiment, Rsamtools, GenomicAlignments, rtracklayer
+        GenomeInfoDb, SummarizedExperiment, Rsamtools, GenomicAlignments, rtracklayer
 Suggests: biomaRt, ChIPpeakAnno, parallel, GenomicFeatures
 License: GPL-2
 URL: http://www.cs.utoronto.ca/~yueli/software.html
diff --git a/NAMESPACE b/NAMESPACE
index a419d73..ab7e47e 100644
--- a/NAMESPACE
+++ b/NAMESPACE
@@ -1,4 +1,4 @@
-import(methods, S4Vectors, IRanges, GenomicRanges, SummarizedExperiment, Rsamtools, GenomicAlignments)
+import(methods, S4Vectors, IRanges, GenomeInfoDb, GenomicRanges, SummarizedExperiment, Rsamtools, GenomicAlignments)
 
 importMethodsFrom(rtracklayer, browserSession,
                  browserView, export, genome,
